e2e/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "e2e"
+authors = { workspace = true }
+edition = { workspace = true }
+homepage = { workspace = true }
+readme = { workspace = true }
+license = { workspace = true }
+repository = { workspace = true }
+rust-version = { workspace = true }
+version = { workspace = true }
+publish = false
+
+[dev-dependencies]
+# used in test assertions
+chrono = { workspace = true }
+# TODO: [issue](https://github.com/FuelLabs/fuels-rs/issues/1375) needs to be removed, `ScriptTransaction` and `CreateTransaction` in `fuels` use `fuel_tx::Input` but don't reexport or convert it into a `fuels` owned type
+fuel-tx = { workspace = true }
+fuels = { workspace = true }
+# TODO: [issue](https://github.com/FuelLabs/fuels-rs/issues/1376) because setup-program-test uses it but `fuels` did not reexport `rand`
+rand = { workspace = true }
+# used in test assertions
+tai64 = { workspace = true }
+tempfile = { workspace = true }
+tokio = { workspace = true }
+
+[features]
```

Comment:
we probably need to add a `std` or `default` feature that enables `fuels/std`. Additionally, we need to add a `coin-cache` feature because inside the `e2e` we have several tests behind `#[cfg(feature = "coin-cache")]`